src/components/DeveloperCard.vue

```diff
@@ -209,6 +209,16 @@ export default {
     margin-right: 0;
 }
 
+.text-card {
+   overflow: hidden;
+   text-overflow: ellipsis;
+   display: -webkit-box;
+   line-height: 16px;
+   max-height: 64px;
+   -webkit-line-clamp: 4;
+   -webkit-box-orient: vertical;
+}

```

Comment:
Since we're using Sass we can take advantage of the nesting, so add this inside of the [description](https://github.com/VueDominicana/DominicanWhoCodes/pull/53/files#diff-b44af088e47ef6ad179cee8a39a1875bR199) by targeting the span directly.

Reply:
Ok, thanks bro.